### ISSUE TITLE
Request to add Huber loss function

### ISSUE THREAD

**Issue by 8bitmp3** [keras-team/keras#12943]
Request to add a Huber loss function similar to the `tf.keras.losses.Huber` class (TF 2.0 beta [API docs](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/losses/Huber
), [source](https://github.com/tensorflow/tensorflow/blob/9e0f4e9b11329f81ef3ae3bff946bfb04a619b76/tensorflow/python/keras/losses.py
)).

Potential implementation - as per @ageron notebook - but with needed amendments:
```python
def huber_fn(y_true, y_pred):
  error = y_true - y_pred
  is_small_error = tf.abs(error) < 1 # replace `tf` with `K` where `K = keras.backend`
  squared_loss = tf.square(error) / 2 # replace `tf` with `K`
  linear_loss = tf.abs(error) - 0.5 # replace `tf` with `K`
  return tf.where(is_small_error, squared_loss, linear_loss) # replace `tf.where`
```

Huber loss was requested here [#6410](https://github.com/keras-team/keras/pull/6410) but Log-Cosh was introduced instead (2x differentiable). `tf.keras.losses.LogCosh` [also](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/losses/LogCosh) exists.

**Comment by 8bitmp3** [comment]
Thanks @jvishnuvardhan - looks like [`tf.keras.losses.huber`](https://www.tensorflow.org/api_docs/python/tf/keras/losses/huber?hl=en) is now available in `tf-nightly` cc @dynamicwebpaige

**Comment by 8bitmp3** [comment]
Thank you all - looks like [`tf.keras.losses.huber`](https://www.tensorflow.org/api_docs/python/tf/keras/losses/huber?hl=en) is part of TensorFlow 2.3 Core. Closing the issue @jvishnuvardhan @dynamicwebpaige 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/19637339/89465540-3130ac00-d76a-11ea-91a0-9317007d8709.png">
